src/Sylius/Component/Shipping/Resolver/CompositeMethodsResolver.php

```diff
@@ -45,4 +45,12 @@ public function getSupportedMethods(ShippingSubjectInterface $shippingSubject)
 
         return [];
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(ShippingSubjectInterface $subject)
+    {
+        return true;
```

Comment:
@pamil @CarwynNelson the idea was, to unified all shipping methods resolvers with one ``MethodsResolverInterface``, to give possibility to inject both specific method resolver and composite resolver, which contains resolvers registry. Due to that fact, we must keep ``supports()`` method in all of them, as they can be used separately. I agree, that returning ``true`` here is nasty, but I can't came up with better option.